Public/Src/Engine/Scheduler/Filter/ClosureFunctionFilter.cs

```diff
@@ -163,7 +168,7 @@ public enum ClosureMode
         /// <summary>
         /// Gets only the direct neighbors (excludes the matching pips for the inner filter).
         /// NOTE: If pip is a matching neighbor of one of the matching pips for the inner filter
-        /// it will be included.
+        /// it will be excluded.

```

Comment:
>/// it will be excluded. [](start = 8, length = 24)

I hope there's also some user facing documentation to update for this. I know there's a big block of help text for filtering. If requiredfor, copydpt, dpc, dpt, and these special filters aren't already in there, could you add?

Reply:
Added to helptext